calculator.js

```diff
@@ -1,5 +1,54 @@
 const exampleAdditionInput = {
   num1: 3,
-  num2: 5,
-  operation: 'add',
+  num2: 1,
+  operation: '/',
 }
+
+//another way of writing it ...const add = (num1, num2) num1 + num2;
```

Comment:
You missed the `=>` in the function definition.

```suggestion
//another way of writing it ...const add = (num1, num2) => num1 + num2;
```